base/applications/taskmgr/lang/en-US.rc

```diff
@@ -377,6 +377,8 @@ BEGIN
     ID_PROCESS_PAGE_SETPRIORITY_NORMAL "Sets process to the NORMAL priority class"
     ID_PROCESS_PAGE_SETPRIORITY_BELOWNORMAL "Sets process to the BELOW NORMAL priority class"
     ID_PROCESS_PAGE_SETPRIORITY_LOW "Sets process to the LOW priority class"
+    ID_PROCESS_PAGE_PROPERTIES "Shows process executable file properties"
+    ID_PROCESS_PAGE_OPENFILELOCATION "Opens process executable file location"
```

Comment:
"process executable file" (in both strings) --> "process file"

Reply:
OK.